target/linux/octeon/base-files/etc/board.d/01_network

```diff
@@ -8,6 +8,9 @@
 board_config_update
 
 case "$(board_name)" in
+itus_shield-router)
```

Comment:
That's almost what I was looking for, please use the name with comma "," for the board name, as that's what's done for DTS targets (board name/compatible with comma, device definition/image name with underscore):

itus,shield-router